[2023-04-06T09:00:13.433+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: io_resto_casa.respond scheduled__2023-04-06T08:50:00+00:00 [queued]>
[2023-04-06T09:00:13.449+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: io_resto_casa.respond scheduled__2023-04-06T08:50:00+00:00 [queued]>
[2023-04-06T09:00:13.450+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-06T09:00:13.452+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 3
[2023-04-06T09:00:13.453+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-06T09:00:13.476+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): respond> on 2023-04-06 08:50:00+00:00
[2023-04-06T09:00:13.484+0000] {standard_task_runner.py:55} INFO - Started process 9191 to run task
[2023-04-06T09:00:13.489+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'io_resto_casa', 'respond', 'scheduled__2023-04-06T08:50:00+00:00', '--job-id', '178', '--raw', '--subdir', 'DAGS_FOLDER/ciao.py', '--cfg-path', '/tmp/tmp24iqsb8z']
[2023-04-06T09:00:13.494+0000] {standard_task_runner.py:83} INFO - Job 178: Subtask respond
[2023-04-06T09:00:13.605+0000] {task_command.py:389} INFO - Running <TaskInstance: io_resto_casa.respond scheduled__2023-04-06T08:50:00+00:00 [running]> on host cfece00ff772
[2023-04-06T09:00:13.718+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=io_resto_casa
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2023-04-06T08:50:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-06T08:50:00+00:00
[2023-04-06T09:00:13.722+0000] {python.py:177} INFO - Done. Returned value was: Greet Responded Again
[2023-04-06T09:00:13.755+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=io_resto_casa, task_id=respond, execution_date=20230406T085000, start_date=20230406T090013, end_date=20230406T090013
[2023-04-06T09:00:13.823+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-06T09:00:13.851+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
